src/client/datascience/jupyter/kernels/kernel.ts

```diff
@@ -219,8 +220,9 @@ export class Kernel implements IKernel {
         }
         const key = uri.toString();
         if (!this.kernelValidated.get(key)) {
-            this.isRawNotebookSupported =
-                this.isRawNotebookSupported || this.rawNotebookSupported.isSupportedForLocalLaunch();
```

Comment:
This was wrong, if we're using remote connection, we cannot use `raw`

Reply:
This is the bug that is being fixed in this PR.

Reply:
@DonJayamanne Isn't this just the .supported method on the rawNotebookSupported Service?

Reply:
Yes, youre right, we don't need to cache this.